patch/0001-chore-application-id-to-bcsc-dev.patch

```diff
@@ -0,0 +1,40 @@
+From 736596b3fcc6926fc04e0bb2f9e7badec0cfc4b6 Mon Sep 17 00:00:00 2001
+From: "Jason C. Leach" <jason.leach@fullboar.ca>
+Date: Fri, 1 Aug 2025 13:41:45 -0700
+Subject: [PATCH] chore: application id to bcsc dev
+
+Signed-off-by: Jason C. Leach <jason.leach@fullboar.ca>
+---
+ app/android/app/build.gradle         | 2 +-
+ app/android/app/google-services.json | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/app/android/app/build.gradle b/app/android/app/build.gradle
+index 17d1efc6..a2fe13fc 100644
+--- a/app/android/app/build.gradle
++++ b/app/android/app/build.gradle
+@@ -95,7 +95,7 @@ android {
+     }
+ 
+     defaultConfig {
+-        applicationId "ca.bc.gov.BCWallet"
++        applicationId "ca.bc.gov.id.servicescard.dev"
+         minSdkVersion rootProject.ext.minSdkVersion
+         targetSdkVersion rootProject.ext.targetSdkVersion
+         versionCode System.getenv("VERSION_CODE") ? System.getenv("VERSION_CODE").toInteger() : 1
+diff --git a/app/android/app/google-services.json b/app/android/app/google-services.json
+index a3a58790..13071b8f 100644
+--- a/app/android/app/google-services.json
++++ b/app/android/app/google-services.json
```

Comment:
I am wondering if we need to make the content of this file configurable. We might need to add BCSC bundle ids to our existing firebase account. The docs recommends to download the file from the console:
https://firebase.google.com/docs/android/setup#add-config-file

Reply:
@cvarjao The default one on the file system is the BC Wallet Firebase config and the patch just converts it to the one from BCSC to use that firebase when in Single App mode. I can either patch or just overwrite during the build. Both options are ok with me.